src/OrchardCore.Modules/OrchardCore.Workflows/OrchardCore.Workflows.csproj

```diff
@@ -27,7 +27,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Liquid.Abstractions\OrchardCore.Liquid.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Localization.Abstractions\OrchardCore.Localization.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Module.Targets\OrchardCore.Module.Targets.csproj" />
-    <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />
```

Comment:
Don't remove this reference. It's for the tag helpers and needed to work in production.

Reply:
ResourceManagement reference added back.